### PR TITLE
chore: clean examples

### PR DESCRIPTION
**Pull request by zfields**
- Remove/replace deprecated API calls
- Fix whitespace

### CODE REVIEW

**Comment by zfields** [comment]
Rebased to consider latest merge

File: examples/ArduinoIoTCloud-Schedule/thingProperties.h [hunk 1]
```diff
@@ -28,16 +29,16 @@ void initProperties() {
...
-  ArduinoCloud.addProperty(monthly, READWRITE, ON_CHANGE);
-  ArduinoCloud.addProperty(yearly, READWRITE, ON_CHANGE);
+  ArduinoCloud.addProperty(switchButton, Permission::Write).onUpdate(nullptr);
+  ArduinoCloud.addProperty(oneShot, Permission::ReadWrite).onUpdate(nullptr);
+  ArduinoCloud.addProperty(minute, Permission::ReadWrite).onUpdate(nullptr);
+  ArduinoCloud.addProperty(hourly, Permission::ReadWrite).onUpdate(nullptr);
+  ArduinoCloud.addProperty(daily, Permission::ReadWrite).onUpdate(nullptr);
+  ArduinoCloud.addProperty(weekly, Permission::ReadWrite).onUpdate(nullptr);
+  ArduinoCloud.addProperty(monthly, Permission::ReadWrite).onUpdate(nullptr);
+  ArduinoCloud.addProperty(yearly, Permission::ReadWrite).onUpdate(nullptr);
```
Comment (pennam) [inline examples/ArduinoIoTCloud-Schedule/thingProperties.h:39]:
```suggestion
  ArduinoCloud.addProperty(switchButton, Permission::Write);
  ArduinoCloud.addProperty(oneShot, Permission::ReadWrite);
  ArduinoCloud.addProperty(minute, Permission::ReadWrite);
  ArduinoCloud.addProperty(hourly, Permission::ReadWrite);
  ArduinoCloud.addProperty(daily, Permission::ReadWrite);
  ArduinoCloud.addProperty(weekly, Permission::ReadWrite);
  ArduinoCloud.addProperty(monthly, Permission::ReadWrite);
  ArduinoCloud.addProperty(yearly, Permission::ReadWrite);
```

File: examples/ArduinoIoTCloud-Schedule/thingProperties.h [hunk 2]
```diff
@@ -28,16 +29,16 @@ void initProperties() {
...
 #elif defined(BOARD_HAS_LORA)
-  ArduinoCloud.addProperty(switchButton, 1, WRITE, ON_CHANGE);
+  ArduinoCloud.addProperty(switchButton, 1, Permission::Write).onUpdate(nullptr);
```
Comment (pennam) [inline examples/ArduinoIoTCloud-Schedule/thingProperties.h:41]:
```suggestion
  ArduinoCloud.addProperty(switchButton, 1, Permission::Write);
```

**Review by pennam — CHANGES_REQUESTED** [review]
@zfields I get your point, but I would love to keep the include of the example as equal as possible to the examples generated from the cloud editor. We have two differences:
 - In #412 we have moved the `#include "arduino_secrets.h"` from the .ino to the `thingProperties.h` file
 - We kept the `#include <Arduino_ConnectionHandler.h>` in the `arduino_secrets.h` file because we need to know which secrets to include.
 
Moreover the `AIoTC_Config.h` is somehow an internal configuration file and was not really ment to be included in the sketch files.

**Comment by zfields** [comment]
> Moreover the `AIoTC_Config.h` is somehow an internal configuration file and was not really meant to be included in the sketch files.

It appears the internal configuration is already spilling out into the example. `arduino_secrets.h` is using `#define` variables that originate from `AIoTC_Config.h` (e.g. `BOARD_HAS_WIFI`, `BOARD_HAS_SECRET_KEY`, `BOARD_HAS_GSM`, `BOARD_HAS_NB`, `BOARD_HAS_CATM1_NBIOT`, `BOARD_HAS_LORA`, `BOARD_HAS_ETHERNET`).

I agree that parity between the cloud generated examples and the examples in the library is of the utmost importance. I thought changing it here would also change it there. I will change back the headers to match the cloud.

My only feedback, is that it is currently rather confusing to understand where variables are coming from, and how things are being put together.

**Comment by zfields** [comment]
@pennam I've implemented your requested changes and rebased on the latest release (0.15.1). :+1:

**Review by pennam — APPROVED** [review]
:+1:

**Comment by zfields** [comment]
rebased on `master`

**Pull request merged**